juntagrico/views.py

```diff
@@ -122,28 +122,39 @@ def job(request, job_id):
     '''
     member = request.user.member
     job = get_object_or_404(Job, id=int(job_id))
+    slotrange = list(range(0, job.slots))
+    allowed_additional_participants = list(
+        range(1, job.free_slots + 1))
+    job_fully_booked = len(allowed_additional_participants) == 0
+    job_is_in_past = job.end_time() < timezone.now()
+    job_is_running = job.start_time() < timezone.now()
+    job_canceled = job.canceled
+    can_subscribe = job.infinite_slots or not (
+        job_fully_booked or job_is_in_past or job_is_running or job_canceled)
+    renderdict = get_menu_dict(request)
 
     if request.method == 'POST':
```

Comment:
instead of two if inside each other merge them
if request.method == 'POST' and can_subscribe and 0<int(request.Post['jobs'])<=free_slots:
reduces cyclomatic complexity

Reply:
Fixed in 85abb8d